pontoon/base/models.py

```diff
@@ -1902,7 +1902,7 @@ def for_project_locale(self, project, locale, paths=None, statuses=None,
         # Temporary optimization: use simpler status & extra filter DB queries for non-PO projects,
         # which don't store translations for different plural forms in separate Translation model
         # instances.
-        no_plurals = True if len(project.resources.filter(format='po')) == 0 else False
+        no_plurals = project.resources.filter(format='po').exists()
```

Comment:
I think you meant 
```python
no_plurals = not project.resources.filter(format='po').exists()
```

Reply:
And I even updated test to compensate for that! Grrrr... (and thanks for the catch!)